openrtb_ext/imp_adnuntius.go

```diff
@@ -5,4 +5,5 @@ type ImpExtAdnunitus struct {
 	Network   string `json:"network"`
 	NoCookies bool   `json:"noCookies"`
 	MaxDeals  int    `json:"maxDeals"`
+	PriceType string `json:"priceType,omitempty"`
```

Comment:
should also update 
https://github.com/prebid/prebid-server/blob/71f76eb39e23d778e7ff3356711004775851e56e/static/bidder-params/adnuntius.json

to include `priceType` as optional param